source/util/AMTimer.h

```diff
@@ -0,0 +1,160 @@
+#ifndef AMTIMER_H
+#define AMTIMER_H
+
+#include <QTimer>
+#include <QTime>
+/*!
+  * A class which provides a wrapper of a QTimer, adding the ability to pause/
+  * unpause, with setPaused(bool) and to find out how long time timer has been
+  * running with elapsedTime().
+  */
+class AMTimer : public QObject
```

Comment:
When I was working on the functions `isActive()`, `isRunning()`, `isPaused()`, I found that it is easier/clearer to encapsulate a QTimer than inherit from it.